src/platform.ml

```diff
@@ -58,9 +58,35 @@ module type THREAD = sig
   (** Re-schedule the calling thread without suspending it. *)
 end
 
+module type RAW_STATS = sig
+  type t = {
+    mutable bytes_read : int;
+    mutable nb_reads : int;
+    mutable bytes_written : int;
+    mutable nb_writes : int;
+  }
+
+  val fresh_stats : unit -> t
+  val reset : t -> unit
+end
+
+module type IO_STATS = sig
+  include RAW_STATS
+
+  val get : unit -> t
+  val get_by_file : string -> t
+  val get_all : unit -> (string * t) list
+  val reset : unit -> unit
```

Comment:
You could rename that one to `reset_all` to avoid shadowing `val fresh_stats : unit -> t`